cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/GDPRUserConsent.java

```diff
@@ -13,13 +13,13 @@
     public JSONObject jsonConsents;
     public String consentString;
 
-    public GDPRUserConsent(){
-        acceptedVendors = new ArrayList<>();
-        acceptedCategories = new ArrayList<>();
-        consentString = "";
+    private static JSONObject emptyConsents = null;
+
+    public GDPRUserConsent() throws JSONException {
```

Comment:
We should not use JSONObject here, we can just set the jsonConsent field directly, no exception needed!

Reply:
we need exception here , as we are putting empty values in jsonConsent object , otherwise we will get exception in application.